data/json/items/melee.json

```diff
@@ -1411,7 +1411,44 @@
         "to_hit": 2,
         "techniques" : ["PRECISE", "RAPID", "WBLOCK_2"],
         "flags": ["STAB", "SHEATH_SWORD"]
-    },{
+    },
+  {
+    "id": "sword_cane",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "brown",
+    "name": "sword cane (sheathed)",
+    "description": "A cane containing a hidden blade.  The sword cane was a popular fashion accessory for the wealthy during the 18th and 19th centuries.",
+    "price": 2000,
+    "price_postapoc": 1000,
+    "material": [ "steel", "wood" ],
+    "weight": 650,
+    "volume": "6 L",
+    "bashing": 3,
+    "to_hit": 0,
+    "use_action": { "type": "transform", "target": "sword_cane_on", "msg": "You unsheath your sword cane." },
+    "techniques": [ "PRECISE", "RAPID", "WBLOCK_2" ]
+  },
+  {
+    "id": "sword_cane_on",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "sword cane",
+    "description": "A cane containing a hidden blade.  The sword cane was a popular fashion accessory for the wealthy during the 18th and 19th centuries.",
```

Comment:
Should not the description state that it is the sword part? And that it can be activated to sheaf? In fact, shouldn't the activation of "off" turn it into two items, sheaf and sword?

Reply:
>Should not the description state that it is the sword part?
What do you mean?

I'm pretty sure "use_action" does not support creating two items.

Reply:
When you have activated the cane to expose the blade, the description of "A cane containing a hidden blade" no longer seems a good description.  Maybe something along like "A thin rapier like sword that can be hidden inside the hollow cane part you also have." Ok that is a terrible sentence, but maybe you see what I mean?

Reply:
Sure ^^